tutorials-teaming-in-litmus/index.html

```diff
@@ -106,12 +106,6 @@ <h2 is-upgraded>Editing an invitation</h2>
 <p>In an event that you want to revoke a sent invitation, or edit the roles set for a specific user when the initial invitation was sent, the portal allows you to edit the invitation already sent to users. </p>
 <p>Navigate to the invited tab, here you would be able to change the role of an invited user or delete the user from the list of invitees by clicking the basket button. </p>
 <p class="image-container"><img style="width: 624.00px" src="img/38747f44089d9bfb.png"></p>
-<h2 is-upgraded>Accepting an invitation</h2>
```

Comment:
Why are we removing this?

Reply:
This was not intentionally removed. 

I mentioned [here](https://github.com/litmuschaos/tutorials/pull/23#issuecomment-968424100) that all contributions made should have a single source of truth(Google docs) since the mark up is auto generated. 

That particular contribution was done by editing the auto-generated code, which is why it was no longer there when I regenerated since it wasn't an update to Google Docs(The single source of truth). 

I'll add it to the docs and regenerate now. @Jonsy13

Reply:
Fixed @Jonsy13

Reply:
Hi @Jonsy13. Have you had time to take a final look?